ui/media/js/main.js

```diff
@@ -849,6 +851,9 @@ function abortTask(task) {
             console.error(e)
         }
     })
+    if (task.batchesDone > 0) {
+        document.title = "Stopped - Easy Diffusion"
```

Comment:
abortTask will be called by a user, so we don't need to say Stopped in the title bar (the user will already be in the ED tab). I think we can set the title back to "Easy Diffusion".

Reply:
Applied the change in [ec7553f](https://github.com/easydiffusion/easydiffusion/pull/1377/commits/ec7553fe526bb488485067226b4283499e6eef86), but should I change "Completed - Easy Diffusion" to just "Easy Diffusion" too then? Since it would be clear that if there are no more images that it is finished.